source/_integrations/squeezebox.markdown

```diff
@@ -12,10 +12,13 @@ ha_config_flow: true
 ha_dhcp: true
 ha_platforms:
   - media_player
+  - binary_sensor
 ha_integration_type: integration
 ---
 
-The Squeezebox integration allows you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was previously known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
+The Squeezebox integration allows you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was formerly known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
+
```

Comment:
Further down, before the Action section, please add a section called ## Entities, ### Binary sensors, listing the binary sensors together with a description.